frontend/server/src/main/java/org/pytorch/serve/wlm/WorkLoadManager.java

```diff
@@ -229,12 +229,10 @@ private void addThreads(
 
             BatchAggregator aggregator;
 
-            if (model.isSequenceBatching()) {
-                if (model.isSequenceContinuousBatch()) {
-                    aggregator = new SequenceContinuousBatching(model);
-                } else {
-                    aggregator = new SequenceBatching(model);
-                }
+            if (model.isSequenceContinuousBatch()) {
```

Comment:
This is incorrect as it tests for isSequenceContinuousBatch in both cases. I also meant that we should not need to create the separate configuration keyword sequenceContinuousBatching but we can just rely on sequenceBatching + continuousBatching being set to True.

Reply:
remove model config "sequenceContinuousBatching"